charged-ieee/lib.typ

```diff
@@ -131,7 +147,7 @@
       columns: slice.len() * (1fr,),
       gutter: 12pt,
       ..slice.map(author => align(center, {
-        text(12pt, author.name)
```

Comment:
The author names are too small. They are that small neither in the [template you linked](https://ctan.joethei.xyz/macros/latex/contrib/IEEEtran/IEEEtran_HOWTO.pdf) nor in [Overleaf's template](https://www.overleaf.com/latex/templates/ieee-conference-template/grfzhhncsfqn.pdf).

Reply:
You're right my bad, I got this one from the [.doc](https://www.ieee.org/conferences/publishing/templates.html).